README.md

```diff
@@ -6,48 +6,55 @@
 </h3>
 
 <p align="center">
-    <a href="https://github.com/catppuccin/fish/stargazers"><img src="https://img.shields.io/github/stars/catppuccin/fish?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
-    <a href="https://github.com/catppuccin/fish/issues"><img src="https://img.shields.io/github/issues/catppuccin/fish?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
-    <a href="https://github.com/catppuccin/fish/contributors"><img src="https://img.shields.io/github/contributors/catppuccin/fish?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
+	<a href="https://github.com/catppuccin/fish/stargazers"><img src="https://img.shields.io/github/stars/catppuccin/fish?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
+	<a href="https://github.com/catppuccin/fish/issues"><img src="https://img.shields.io/github/issues/catppuccin/fish?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
+	<a href="https://github.com/catppuccin/fish/contributors"><img src="https://img.shields.io/github/contributors/catppuccin/fish?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
 </p>
 
 <p align="center">
-  <img src="https://raw.githubusercontent.com/catppuccin/fish/main/assets/screenshot.webp"/>
+	<img src="assets/preview.webp"/>
 </p>
 
-## Usage
+## Previews
+
+<details>
+<summary>🌻 Latte</summary>
+<img src="assets/latte.webp"/>
+</details>
+<details>
+<summary>🪴 Frappé</summary>
+<img src="assets/frappe.webp"/>
+</details>
+<details>
+<summary>🌺 Macchiato</summary>
+<img src="assets/macchiato.webp"/>
+</details>
+<details>
+<summary>🌿 Mocha</summary>
+<img src="assets/mocha.webp"/>
+</details>
 
-### 1. Install it
+## Usage
 
-#### Plugin Managers:
+1. Install with a plugin manager (see below) or manually by placing the theme file of your choice from [`themes/`](./themes/) under `~/.config/fish/themes/`.
 
-+ [Fisher](https://github.com/jorgebucaran/fisher)
+- [Fisher](https://github.com/jorgebucaran/fisher)
 
 ```sh
 fisher install catppuccin/fish
 ```
 
-+ [Oh My Fish](https://github.com/oh-my-fish/oh-my-fish)
+- [Oh My Fish](https://github.com/oh-my-fish/oh-my-fish)
 
 ```sh
 omf install https://github.com/catppuccin/fish
 ```
 
-#### Manually:
```

Comment:
Any particular reason we don't offer this method anymore?

Reply:
Still offered. Just mentioned earlier in the guide.

Reply:
Oh, I skimmed over it by accident, all good then :+1: